app/controllers/admin/base_controller.rb

```diff
@@ -35,5 +37,15 @@ def are_appointments_enabled?
         render_not_found
       end
     end
+
+    def flash_highlight(id_or_object)
+      identifier = if String === id_or_object
```

Comment:
```suggestion
      identifier = if id_or_object.is_a?(String)
```

Not a serious change or anything, I just find the `is_a?` to be a bit more readable than `===` (there might be some subtle difference in behavior but I don't think it would impact this)

Reply:
also works: 

```rb
case id_or_object
when String
  #...
else
  #...
end
```

I guess the real lesson is that the `===` feels odd to me, but I'm pretty sure that's just a me thing 😆